SpatialGDK/Source/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1176,3 +1285,9 @@ USpatialActorChannel* USpatialNetDriver::GetActorChannelByEntityId(Worker_Entity
 {
 	return EntityToActorChannel.FindRef(EntityId);
 }
+
+void USpatialNetDriver::WipeWorld(const USpatialNetDriver::ServerTravelDelegate& LoadSnapshotAfterWorldWipe)
+{
+	UE_LOG(LogSpatialOSNetDriver, Error, TEXT("Wiping world!"));
```

Comment:
This is not an `Error` right? `Warning` would be fair. And just saying "Wiping world" lacks too much context for log.

Reply:
The logs need a pass since I left them like this for debugging purposes whilst making changes. Updating now.